apps/typeform/frontend/src/processTokenCallback.ts

```diff
@@ -0,0 +1,25 @@
+export default (window: Window) => {
+  const { searchParams, search } = new URL(window.location.href);
+
+  if (search.length) {
+    const error = searchParams.get('error');
+
+    if (error) {
+      window.opener.postMessage({ error }, '*');
+      return;
+    }
+
+    const token = searchParams.get('token') || '';
+    const expiresIn = parseInt(searchParams.get('expiresIn') || '', 10) || 3200;
```

Comment:
Did this `3200` default come from typeform?

Reply:
Changed it to use the default Typeform token expiration and refactor it to a const